packages/plugin-betswirl/src/types.ts

```diff
@@ -0,0 +1,31 @@
+import { z } from "zod";
+import { CASINO_GAME_TYPE, maxGameBetCountByType } from "@betswirl/sdk-core";
+
+export const hexAddress = z
+    .string()
+    .regex(/^0x[a-fA-F0-9]{40}$/, "The address must be a valid EVM address");
+
+export const casinoBetParams = {
+    betAmount: z.string().describe("The bet amount"),
+    token: z
+        .string()
+        .describe("The token to bet with"),
```

Comment:
Make this both nullable as well

```suggestion
    betAmount: z.string().describe("The bet amount").nullable(),
    token: z
        .string()
        .nullable()
        .describe("The token to bet with"),
```